src/extension/sldService/src/main/java/org/geoserver/sldservice/utils/classifier/RasterSymbolizerBuilder.java

```diff
@@ -264,6 +264,38 @@ public ColorMap jenksClassification(
         return getColorMapFromBreaks(breaks, open, continuous, percentages);
     }
 
+    /**
+     * Builds a {@link ColorMap} based on equal intervals between the min and max value found in the
+     * raster
+     *
+     * @param image The source image
+     * @param intervals Number of resulting intervals
+     * @param continuous If the resulting ColorMap should be of type interval (discrete) or of type
+     */
+    public ColorMap standardDeviationClassification(
+            RenderedImage image, int intervals, boolean open, boolean continuous) {
+        ImageWorker iw = getImageWorker(image);
+        Statistics[][] stats = getStatistics(iw);
+        double mean = (double) stats[0][0].getResult();
+        double stddev = (double) stats[0][1].getResult();
+        double[] extrema = (double[]) stats[0][2].getResult();
+        double min = extrema[0];
+        double max = extrema[1];
+        Number[] breaks = new Number[continuous ? intervals : intervals + 1];
+        for (int i = 0; i < breaks.length; i++) {
+            if (i == 0) breaks[0] = Double.NEGATIVE_INFINITY;
+            else if (i == breaks.length - 1) breaks[breaks.length - 1] = Double.POSITIVE_INFINITY;
```

Comment:
Looking at the code, it may seem putting infinity here is what triggered a bunch of other changes. Is there no other classification method that generates open ended intervals? I would be surprised if there wasn't?

Reply:
Also, I don't see a handling of open vs closed intervals. (parameter "open")